### PR TITLE
Set up new classifier routes for workflows and subject sets

### PR DESCRIPTION
**Pull request by eatyourgreens**
Stubs out new Classify pages for specific workflows and subject sets, as per [ADR 18](https://github.com/zooniverse/front-end-monorepo/blob/master/docs/arch/adr-18.md). Workflow ID and subject set ID are passed down, from the URL, to the classifier component as props.

Package:
app-project

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by shaunanoordin** [comment]
I'm currently testing this on a staging project with multiple workflows, multiple subject sets

```
Staging: /projects/darkeshard/engaging-the-crowds
Project ID: 1922

Workflows:
- 3425 Single Question Workflow
- 3426 Dropdown Workflow

Subject Sets:
- 4803 Transformers images
- 4804 Ultraman images
- 4805 Overwatch images
```

**Comment by shaunanoordin** [comment]
Current observations:

- 👍 The new `.../classifiy/workflow/1234` URLs are properly established (i.e. no 404s)
- 👎 The new URLs still don't allow selecting a specific URL
  - e.g. `http://localhost:3000/projects/darkeshard/engaging-the-crowds/classify/workflow/3427` still shows the default single question workflow (3425).
- The classic query URL does correctly allow selecting a custom URL, so the Classifier bits are working
  - e.g. `http://localhost:3000/projects/darkeshard/engaging-the-crowds/classify?workflow=3427` shows the correct drawing workflow

Testing continues. I need to double check my assumption that the new URLs are/should be passing the workflow IDs down to the Classifier component.

File: packages/app-project/pages/projects/[owner]/[project]/classify/workflow/[workflowID]/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export { default } from '@screens/ClassifyPage'
+
+export function getServerSideProps({ params, req, resp }) {
```
Comment (eatyourgreens) [inline packages/app-project/pages/projects/[owner]/[project]/classify/workflow/[workflowID]/index.js:3]:
```suggestion
export function getServerSideProps({ params, req, res }) {
```
See https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering

File: packages/app-project/pages/projects/[owner]/[project]/classify/workflow/[workflowID]/subject-set/[subjectSetID]/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export { default } from '@screens/ClassifyPage'
+
+export function getServerSideProps({ params, req, resp }) {
```
Comment (eatyourgreens) [inline packages/app-project/pages/projects/[owner]/[project]/classify/workflow/[workflowID]/subject-set/[subjectSetID]/index.js:3]:
```suggestion
export function getServerSideProps({ params, req, res }) {
```
https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering

**Comment by shaunanoordin** [comment]
Side note: I'm just noticing that the command window running `$ app-project: yarn dev` has the following message:

`Initialising Sentry: undefined production (beep boop string of hex characters)`

I hope my localhost dev isn't sending error reports to production Sentry.

**Comment by eatyourgreens** [comment]
>I hope my localhost dev isn't sending error reports to production Sentry.

Only if you've set up the `SENTRY_DSN` environment variable, which I think is the `undefined` in the startup log.

**Comment by shaunanoordin** [comment]
Update: the `WorkflowID` param (from the URL path) IS being properly passed to `<Classifier />` in `ClassifierWrapperContainer.js` as a prop. 👍 This means that the problem is probably Classifier isn't reading the props props as I assumed, which is probably a separate issue from this PR.

In that case, this PR is looking good. Let me wrap up some final tests with Subject Sets.

**Pull request merged**